APCAppCore/APCAppCore/UI/Onboarding/SignUp/APCInEligibleViewController.m

```diff
@@ -36,6 +36,7 @@
 #import "APCOnboardingManager.h"
 #import "APCLog.h"
 #import "APCCustomBackButton.h"
+#import "APCAppDelegate.h"
```

Comment:
This dependency can be avoided as per my other comment (making `showShareAppInOnboarding` a property of the onboarding manager).